BLSQ/iaso
# IA-2802 fix migration of polio as default campaign type

Fixing multiple problems related to migrations linked to Polio Campaign Types (applying the campaign type of Polio to all the previous campaigns by default was not working)

Also adding a slug to Campaign type so that the frontend will have it easier to always set polio as default for campaign type based on a fixed slug 'polio' instead of requiring a DB call to get the id of the campaign type using its name 

Related JIRA tickets : IA-2802

## Self proofreading checklist



## Changes

- Add a slug to Campaign type which is automatically computed from the name.
- Add related migrations to make the slug unique
- Add migration to fix default campaign type to Polio for pre-existing campaigns

## How to test

- You should be able to filter campaign types API using the slug
- You should be able to apply migrations and have all your old Polio campaigns in your local DB to receive the correct campaign type of 'Polio'


## Notes

This should actually be a POLIO ticket but ...
